text/0000-roadmap-2021.md

```diff
@@ -0,0 +1,215 @@
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+The focus of this year is on project health, specifically as it relates to Rust's governance structure.
+
+* Establishing charters for teams in the Rust project
+* Provide for unified process and vocabulary across the project
+* Creating a single place for tracking a list of ongoing projects
+
+The core team will drive these efforts in coordination with all Rust teams and working groups. As the Rust teams have grown over the years, there is increasingly less cross-chatter that naturally happens due to shared membership of teams, and this RFC aims to improve our written documentation to provide for a smoother onboarding process for new team members, as well as improving the ability of current team members to evaluate who to consult on new ideas or projects.
+
+# Motivation
+[motivation]: #motivation
+
+The Rust project governance has grown into a large and diverse structure, with most teams not having well defined expectations about scope and membership. As the project has grown, we have also struggled with a unified place for new and existing contributors to learn about active efforts and join in. Our efforts to establish charters and unified vocabulary will help contributors new and old to navigate our organizational structure. Our expectation is that this work will also help each team better understand what their aims are and will help the core team better serve the team's needs.
+
+# Explanation
+[explanation]: #explanation
+
+## Chartering work in the Rust project
+
+The Rust teams, in concert with the core team, will work to establish a charter for each of the Rust teams over the course of the year, with an aim for defining, particularly, the purpose and membership requirements. Our goal is that going into 2022, all active groups within the Rust project will have well-defined charters and membership. This includes all pre-existing groups in Rust governance, including domain working groups, project groups, and working groups.
+
+We have a number of active efforts in the Rust project which are not currently called teams. By the end of the year, we expect that regardless of the naming, every active group in Rust’s governance will be chartered. The core team will additionally propose an RFC clarifying the naming and setting policy guidelines for team formation towards the end of the year, once we have worked through a number of charters and can set these expectations better.
+
+This RFC does not aim to provide a fine-grained framework for precisely what form these charters will take. Furthermore, it is our expectation that these charters will be “living” documents, parts of which will be updated as the group changes roles or membership, and the precise expectations of charters are likely to evolve over time.
+
+However, at minimum, they should provide answers to these questions:
+
+* What does this group do?
```

Comment:
Do we want to track whether a group exists within another or at the "top-level"?

Reply:
So, my current thinking is that top-level teams will roughly have a folder within the github repo where these charters will be held, and we'll then place subteams within folders (potentially more than one level of nesting, I guess). The charter for the top-level team should also explicitly have a list of subteams under it, mostly so that there's a clear listing on a diff level - it's also possible the folder structure won't work out and this would be the alternative. I would expect subteam charters to make it pretty clear who their parent team(s) are.